ouroboros-network/src/Ouroboros/Network/Protocol/TxSubmission/Type.hs

```diff
@@ -228,7 +228,7 @@ instance Protocol (TxSubmission txid tx) where
 
   exclusionLemma_NobodyAndServerHaveAgency TokDone tok = case tok of {}
 
-{-# WARNING MsgKThxBye "MsgKThxBye: should only be used with not yet introduced NodeToNodeV_2" #-}
+{-# WARNING MsgKThxBye "MsgKThxBye: should only be used with not yet introduced NodeToNodeV_3" #-}
```

Comment:
I would rather change this to: `MsgKThxBye`: must not be used with `NodeToNodeV_1`. `tx-sumbission` can take care of it for some time now, it's only matter for backward compatibility against a node that was build before introducing it.

Reply:
Ah, I did not know that. Will update.

Reply:
The warning is only left to remind us this when we will start using `MsgKThxBye`.